packages/openactive-integration-tests/test/reporter.js

```diff
@@ -179,19 +179,28 @@ class Reporter {
         const html = await certificationWriter.generateCertificate();
 
         const validationResult = await validateCertificateHtml(html, CONFORMANCE_CERTIFICATE_ID, certificationWriter.awardedTo.name);
-        if (!validationResult || !validationResult.valid) {
+        /* process.env.DEBUG_SAVE_INVALID_CONFORMANCE_CERTIFICATE can be used to ensure a certificate is saved even if it
+        is not valid. This can help with debugging conformance certificates */
+        if (validationResult?.valid || process.env.DEBUG_SAVE_INVALID_CONFORMANCE_CERTIFICATE === 'true') {
+          const filename = 'index.html';
+          await mkdirp(certificationWriter.certificationOutputPath);
+          await fs.writeFile(certificationWriter.certificationOutputPath + filename, html);
+          if (validationResult?.valid) {
+            console.log(`\n${chalk.green(
+              `Conformance certificate for '${certificationWriter.awardedTo.name}' generated successfully: ${certificationWriter.certificationOutputPath + filename} and must be made available at '${CONFORMANCE_CERTIFICATE_ID}' to be valid.`,
+            )}`);
+          } else {
+            console.log(`\n${chalk.red(
+              `Conformance certificate not valid but saved anyway to: ${certificationWriter.certificationOutputPath + filename}`,
```

Comment:
Perhaps better not to leave the code for generating invalid certificates in here, as is open to abuse?

Reply:
Is there another way to debug certificate validation without having to fully run all the tests in debug mode?

Reply:
Also, what kind of abuse do you mean @nickevansuk ? The code is open source anyway

Reply:
Good point!